src/chatlog/chatmessage.cpp

```diff
@@ -234,5 +320,5 @@ QString ChatMessage::detectQuotes(const QString& str, MessageType type)
 
 QString ChatMessage::wrapDiv(const QString &str, const QString &div)
 {
-    return QString("<div class=%1>%2</div>").arg(div, str);
+    return QString("<p class=%1>%2</p>").arg(div, /*QChar(0x200E) + */str);
```

Comment:
Actually, from what @TheSpiritXIII said on IRC, this was commented out to test whether it would fix font(?) issue that @towlie was experiencing on windows, but since it didn't help, it should be uncommented, since it fixes trimmed spaces in chatform.